cl/compile_test.go

```diff
@@ -3211,3 +3211,17 @@ func main() {
 }
 `)
 }
+
+func TestGoInstr(t *testing.T) {
+	gopClTest(t, `package main
+
+//go:noinline
```

Comment:
since you are testing the parser, it might be a good idea to add a case where go:something is not at the beginning for the comment, like // followed by the space, or end of the line comment with //go:something, or it is a the beginning of /* comment etc.

Reply:
thx